noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/components/private_kernel_circuit_output_validator.nr

```diff
@@ -85,10 +85,6 @@ impl PrivateKernelCircuitOutputValidator {
             private_call.historical_header,
             "mismatch historical_header",
         );
-        assert(
-            is_empty(self.output.constants.global_variables),
```

Comment:
Removed this check as a new struct `TxConstantData` is now used in private land, which includes all fields in `CombinedConstantData` except for `global_variables`.